tensorflow/compiler/jit/BUILD

```diff
@@ -18,9 +18,21 @@ package(
     default_visibility = [":internal"],
 )
 
+load("//tensorflow:tensorflow.bzl", "cc_header_only_library")
 load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
 
+cc_header_only_library(
```

Comment:
Perhaps add a comment explaining the purpose of this library. (This is mostly so people don't delete it because it looks unused.

I would be tempted to remove the `_lib` suffix from the name, too.